### PR TITLE
no create metadata on edit

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
[Ticket-  TBD](linkToTicket)

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
`convert_create_agenda_item_query` used by both `create_agenda_item` and `edit_agenda_item`

**Comment by elizabeth-jimenez** [comment]
question: not super familiar on what all data is required for a successful create and edit of AI, so not super confident on this PR

File: talentmap_api/fsbid/services/agenda.py [hunk 1]
```diff
@@ -723,6 +725,12 @@ def convert_create_agenda_item_query(query):
         "aiseqnumref": None,
         "aiitemcreatorid": user_id,
     }
+
+    if aiseqnum:
+        del q['aicreateid']
+        del q['aicreatedate']
+        del q['aiitemcreatorid']
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda.py:732]: may need to grab it and pass it along, instead, but i think it would be a diff approach to hruid

Comment (scott062) [reply talentmap_api/fsbid/services/agenda.py:732]: Prob just grab it out of ref data - but i think you're right. We are overwriting it on subsequent edits

File: talentmap_api/fsbid/services/agenda.py [hunk 2]
```diff
@@ -726,16 +726,45 @@ def convert_create_agenda_item_query(query):
...
-        del q['aicreateid']
-        del q['aicreatedate']
-        del q['aiitemcreatorid']
-
     logger.info('creating AI query mapping')
     logger.info(q)
     print(q)
     return q
 
+def convert_edit_agenda_item_query(query):
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda.py:734]: 😻

File: talentmap_api/fsbid/services/agenda.py [hunk 3]
```diff
@@ -703,25 +703,29 @@ def convert_create_agenda_item_query(query):
...
     q = {
-        "aipmiseqnum": pydash.get(query, "pmiseqnum", ""),
-        "aiempseqnbr": pydash.get(query, "personId", ""),
-        "aiperdetseqnum": pydash.get(query, "personDetailId", ""),
-        "aiaiscode": pydash.get(query, "agendaStatusCode", ""),
-        "aitodcode": pydash.get(query, "combinedTod", ""),
-        "aicombinedtodmonthsnum": pydash.get(query, "combinedTodMonthsNum", ""),
-        "aicombinedtodothertext": pydash.get(query, "combinedTodOtherText", ""),
-        "aiasgseqnum": pydash.get(query, "assignmentId", ""),
-        "aiasgdrevisionnum": pydash.get(query, "assignmentVersion") or 1,
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda.py:715]: 🗒️

File: talentmap_api/fsbid/services/agenda.py [hunk 4]
```diff
@@ -703,25 +703,29 @@ def convert_create_agenda_item_query(query):
...
+        "aiasgseqnum": query.get("assignmentId", ""),
+        "aiasgdrevisionnum": query.get("assignmentVersion"),
         "aicombinedremarktext": None,
         "aicorrectiontext": None,
         "ailabeltext": None,
         "aisorttext": None,
         "aicreateid": user_id,
         "aicreatedate": None,
         "aiupdateid": user_id,
+        "aiupdatedate": None,
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda.py:724]: 🗒️

Comment (scott062) [reply talentmap_api/fsbid/services/agenda.py:724]: It shouldnt matter either way - DB will set update_date and create_date to system time at time of insert

**Review by scott062 — APPROVED** [review]

File: talentmap_api/fsbid/services/agenda.py (hunk 4 above)
Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/agenda.py:724]: 🖖

**Pull request merged**